src/Avalonia.OpenGL/GlBasicInfoInterface.cs

```diff
@@ -16,9 +16,9 @@ public GlBasicInfoInterface(Func<Utf8Buffer, IntPtr> nativeGetProcAddress) : bas
         {
         }
         
-        public delegate void GlGetIntegerv(int name, out int rv);
-        public delegate IntPtr GlGetString(int v);
-        public delegate IntPtr GlGetStringi(int v, int v1);
+        public delegate void bGlGetIntegerv(uint name, out uint rv);
```

Comment:
Please revert these name changes they bring no benefit and just make it harder to review